newsView.test.js

```diff
@@ -15,4 +15,12 @@ describe('class NewsView', () => {
     view.displayHeadlines();
     expect(document.querySelectorAll('div.headline').length).toEqual(1);
   })
+  it('displayHeadlines shows a picture for each headline on the main page', () => {
```

Comment:
To avoid the creation of all duplicate models with 'new NewsModel()' in each test, you can instruct Jest to create one at the very top, like we did with Rspec. With jest it's beforeEach( () => your initialisation here).